aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -86,88 +51,177 @@ var host = new WebHostBuilder()
+   WebSockets is supported with ASP.NET Core 1.1 or later with the **WebSocket Protocol** role service on Windows Server. For more information, see [WebSockets](xref:fundamentals/websockets).
+
+1. Proceed through the **Confirmation** step to install the web server role and services. A server/IIS restart isn't required after installing the **Web Server (IIS)** role.
+
+**Windows desktop operating systems**
+
+Enable the **IIS Management Console** and **World Wide Web Services**.
+
+1. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen).
+
+1. Open the **Internet Information Services** node. Open the **Web Management Tools** node.
+
+1. Check the box for **IIS Management Console**.
+
+1. Check the box for **World Wide Web Services**.
+
+1. Accept the default features for **World Wide Web Services** or customize the IIS features.
 
-### web.config location
+   **WebSockets**  
+   WebSockets is supported with ASP.NET Core 1.1 or later with the **WebSocket Protocol** app development feature on Windows Desktop. To enable WebSockets, select **World Wide Web Services** > **Application Development Features** > **WebSocket Protocol**. For more information, see [WebSockets](xref:fundamentals/websockets).
 
-.NET Core apps are hosted via a reverse proxy between IIS and the Kestrel server. In order to create the reverse proxy, the *web.config* file must be present at the content root path (typically the app base path) of the deployed app, which is the website physical path provided to IIS. The *web.config* file is required at the root of the app to enable the publishing of multiple apps using Web Deploy.
+1. If the IIS installation requires a restart, restart the system.
 
-Sensitive files exist on the app's physical path, including subfolders, such as *\<assembly_name>.runtimeconfig.json*, *\<assembly_name>.xml* (XML Documentation comments), and *\<assembly_name>.deps.json*. When the *web.config* file is present and configures the site, IIS prevents these sensitive files from being served. **Therefore, it's important that the *web.config* file isn't accidently renamed or removed from the deployment.**
+![IIS Management Console and World Wide Web Services are selected in Windows Features.](index/_static/windows-features-win10.png)
+
+---
+
+## Install the .NET Core Windows Server Hosting bundle
+
+1. Install the [.NET Core Windows Server Hosting bundle](https://aka.ms/dotnetcore-2-windowshosting) on the hosting system. The bundle installs the .NET Core Runtime, .NET Core Library, and the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module). The module creates the reverse proxy between IIS and the Kestrel server. If the system doesn't have an Internet connection, obtain and install the [Microsoft Visual C++ 2015 Redistributable](https://www.microsoft.com/download/details.aspx?id=53840) before installing the .NET Core Windows Server Hosting bundle.
```

Comment:
Note: If the hosting bundle is installed before IIS then you must repair the bundle installation after installing IIS.